osu.Game/Overlays/KeyBindingOverlay.cs

```diff
@@ -12,6 +12,8 @@ namespace osu.Game.Overlays
 {
     public class KeyBindingOverlay : SettingsOverlay
     {
+        protected const float WIDTH = 430;
```

Comment:
why is this required?

Reply:
I needed to make `KeyBindingOverlay` a bit wider to make the `pressAKey` label fit. Otherwise the label text would be truncated at the end.

Reply:
I added this here not to expand the whole `SettingsOverlay`

Reply:
This cannot be done this way.